scripts/scripts.js

```diff
@@ -29,18 +29,18 @@ const CONFIG = {
     ae_en: { ietf: 'en', tk: 'hah7vzn.css' },
```

Comment:
```suggestion
    ae_en: { ietf: 'en', tk: 'hah7vzn.css', base: '' },
```
For English regional locales to appear in getConfig().locale.regions 
and work with MEP Lingo, they require a base property (can be empty string). I've made suggestions in the changed code here, but this would apply to the remaining english regional sites listed here as well: https://wiki.corp.adobe.com/pages/viewpage.action?spaceKey=WP4&title=Project+Lingo+Site+Mapping

cc: @vhargrave @sukamat